apps/dot_ext/forms.py

```diff
@@ -158,9 +166,12 @@ def clean(self):
         cleaned_data = super().clean()
         scope = cleaned_data.get("scope")
 
+        if scope is None:
+            cleaned_data['scope'] = ""
+            scope = ""
+
         # Remove personal information scopes, if requested by bene
         if cleaned_data.get("block_personal_choice"):
             cleaned_data['scope'] = ' '.join([s for s in scope.split(" ")
                                              if s not in settings.BENE_PERSONAL_INFO_SCOPES])
```

Comment:
I'm going to change the naming and logic of the form field `block_personal_choice` to make the code more readable. It especially makes it hard to follow the test cases in combination with the new Application model's `require_demographic_scopes` field.  

I'm thinking about the following and meaning:
KEEP: `Application.require_demographic_scopes` - Does your application need to collect beneficiary demographic information.
CHANGE: `SimpleAllowForm share_demographic_scopes` - Does the beneficiary want to share demographic information.